hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/MapNearCacheInvalidationFromClientTest.java

```diff
@@ -192,7 +192,7 @@ public void run() {
         assertTrueEventually(new AssertTask() {
```

Comment:
If you get a lot of such methods, you could make a method 'assertNullEventually' or assertEqualsEventually.